snow/validators/gvalidators/validator_state_test.go

```diff
@@ -82,7 +82,8 @@ func TestGetMinimumHeight(t *testing.T) {
 	state.server.EXPECT().GetMinimumHeight(gomock.Any()).Return(expectedHeight, errCustom)
 
 	_, err = state.client.GetMinimumHeight(context.Background())
-	require.Error(err)
+	// TODO: require specific error
```

Comment:
Do you want to resolve this now or save it for later?

Reply:
Same for the other ones in this file

Reply:
Fixing the grpc errors is a bigger change... so all of those are left for now intentionally.